sanic_routing/router.py

```diff
@@ -100,6 +100,7 @@ def resolve(
 
         if isinstance(route, RouteGroup):
             try:
+                assert method
```

Comment:
cast is probably better.

Reply:
Another solution is to change Immutable type to ``Dict[Any, Any]``

Reply:
That works for me.